.github/azFunction/AzFunctionInfrastructure/main.bicep

```diff
@@ -0,0 +1,167 @@
+@description('Azure region to deploy in')
```

Comment:
being picky here, but this doesnt comply with the Hungarian naming pattern as per the contribution guide. 

Do you mind updating to match?